server/src/packages/submission/submission.repository.ts

```diff
@@ -1,17 +1,19 @@
 import { Option } from 'funfix';
-import { Uuid } from '../../core';
+import { uuidType } from '../core';
+
+export class SubmissionId extends uuidType<'SubmissionId'>() {}
```

Comment:
Could this `SubmissionId` type be an enum that we keep a list of uuids that we use somewhere in the codebase?

Reply:
I don't understand?

Reply:
sorted f2f